components/eventing-controller/controllers/subscription_controller.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
could we get rid off those boiler plates? we do not have them in other controllers

Reply:
sure, I removed them